client/me/help/help-contact/index.jsx

```diff
@@ -218,6 +222,33 @@ module.exports = React.createClass( {
 		widgetInput.onkeydown( { keyCode: KEY_ENTER } );
 	},
 
+	trackChatDisplayError: function() {
+		var { olark, isChatEnded } = this.state;
+
+		// If the user sent/received a message to/from an operator but the chatbox is not inlined
+		// then something must be going wrong. Lets record this event and give some details
+		if ( this.canShowChatbox() ) {
+			return;
+		}
+
+		const tracksData = {
+			olark_locale: olark.locale,
+			olark_is_chat_ended: isChatEnded,
+			olark_is_ready: olark.isOlarkReady,
+			olark_is_expanded: olark.isOlarkExpanded,
+			olark_is_user_eligible: olark.isUserEligible,
+			olark_is_operator_available: olark.isOperatorAvailable
+		};
+
+		if ( olark.details ) {
+			tracksData.olark_is_conversing = olark.details.isConversing;
+		} else {
+			tracksData.olark_details_missing = true;
+		}
+
+		analytics.tracks.recordEvent( 'calypso_help_contact_chatbox_display_mistake', tracksData );
```

Comment:
I think the verb tends to go last in event names, so maybe something more like `calypso_help_contact_chatbox_mistaken_display`?